Package.swift

```diff
@@ -31,9 +31,7 @@ let package = Package(
   dependencies: [
     .package(
         url: "https://github.com/google/abseil-cpp-binary.git",
-        // TODO(ncooke3): Figure out versioning stratgey for binary Abseil releases.
-        // BEFORE: "0.20220623.0"..<"0.20220624.0"
-        .exact("10.8.0-rc.1")
+        "1.2021110200.0" ..< "1.2021110300.0"
```

Comment:
@paulb777, could you take a close look at this. I tried to copy what was done here: https://github.com/grpc/grpc-ios/blob/main/Package.swift#L23

Reply:
Makes sense. We're allowing both patch updates for the abseil library and for the abseil SPM infra.